src/client/gui/lib/vm_table/zones_dropdown_button.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart' hide Switch;
+import '../switch.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../providers.dart';
+
+class ZonesDropdownButton extends ConsumerWidget {
+  const ZonesDropdownButton({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final zones = ref.watch(zonesProvider);
+    final vmInfos = ref.watch(vmInfosProvider);
+
+    return PopupMenuButton(
+      tooltip: 'Change zone availability',
+      position: PopupMenuPosition.under,
+      child: OutlinedButton.icon(
+        onPressed: null,
+        icon: const Icon(Icons.dns, color: Colors.black, size: 16),
```

Comment:
As mentioned in my other comment, you can get the SVG for the icon from Figma, then this will most likely look like this
```suggestion
        icon: SvgPicture.asset('assets/zones.svg'),
```